collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/RequiredQuestionTest.java

```diff
@@ -72,7 +72,8 @@ public void pressingValidateInOptionsMenuOnDifferentScreen_shouldReturnToQuestio
                 .clickOptionsIcon()
                 .clickOnString(org.odk.collect.strings.R.string.validate)
                 .assertConstraintDisplayed("Custom required message")
-                .assertQuestion("Foo", true);
+                .assertQuestion("Foo", true)
+                .assertQuestion("Bar", true);
```

Comment:
Why do we need the second assertion here?

Reply:
To make sure that both questions are still on one screen. I remember that we had bugs with field-lists that after moving back to a question that needed to be answered or the answer was wrong, that question was no longer displayed in a field-list.

Reply:
Probably worth trying to imply that more in the test title, or add a comment if that's awkward.

Reply:
Done.